packages/@react-types/table/src/index.d.ts

```diff
@@ -10,70 +10,120 @@
  * governing permissions and limitations under the License.
  */
 
-import {AriaLabelingProps, AsyncLoadable, CollectionChildren, DOMProps, MultipleSelection, SectionProps, Sortable, StyleProps} from '@react-types/shared';
+import {AriaLabelingProps, AsyncLoadable, CollectionChildren, DOMProps, MultipleSelection, Sortable, StyleProps} from '@react-types/shared';
 import {GridCollection, GridNode} from '@react-types/grid';
 import {Key, ReactElement, ReactNode} from 'react';
 
 export interface TableProps<T> extends MultipleSelection, Sortable {
-  children: ReactElement<TableHeaderProps<T> | TableBodyProps<T> | SectionProps<T> | RowProps<T>>[],
+  // TODO: I think this shouldn't have Section and row? Also need to fix prop table rendering for this
+  /** The elements that make up the Table. Includes the TableHeader, TableBody, Columns, and Rows. */
+  children: ReactElement<TableHeaderProps<T> | TableBodyProps<T>>[],
+  /** A list of row keys to disable. */
   disabledKeys?: Iterable<Key>
 }
 
 export interface SpectrumTableProps<T> extends TableProps<T>, DOMProps, AriaLabelingProps, StyleProps {
+  /**
+   * Sets the amount of vertical padding within each Table cell.
+   * @default 'regular'
+   */
   density?: 'compact' | 'regular' | 'spacious',
+  /**
+   * Sets the overflow behavior for the Table cell contents.
+   * @default 'truncate'
+   */
   overflowMode?: 'wrap' | 'truncate',
+  /** Whether the Table should be displayed with a quiet style. */
   isQuiet?: boolean,
+  /** Sets what the Table should render when there is no content to display. */
   renderEmptyState?: () => JSX.Element
 }
 
 export interface TableHeaderProps<T> {
+  /** A list of Table columns. */
   columns?: T[],
+  /** A list of `Column(s)` or a function. If the latter, a list of columns must be provided using the `columns` prop. */
   children: ColumnElement<T> | ColumnElement<T>[] | ColumnRenderer<T>
 }
 
 type ColumnElement<T> = ReactElement<ColumnProps<T>>;
 type ColumnRenderer<T> = (item: T) => ColumnElement<T>;
 export interface ColumnProps<T> {
+  /** Rendered contents of the column if `children` contains child columns. */
   title?: ReactNode,
+  /** Static child columns or content to render as the column header. */
   children: ReactNode | ColumnElement<T> | ColumnElement<T>[],
+  /** A list of child columns used when dynamically rendering nested child columns. */
   childColumns?: T[],
-  'aria-label'?: string,
+  // TODO Not sure if this is a thing, doesn't seem to get passed through?
+  /** An accessibility label for the column. */
+  // 'aria-label'?: string,
```

Comment:
Bunch of TODOs in this file where I commented out various props since they don't seem to actually do anything. Not sure if not implemented yet or outdated types.